src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/StartGame.java

```diff
@@ -133,6 +133,10 @@ public StartGame(ConnectionImpl connection) {
 		VarNumberSerializer.writeSVarInt(chunkradius, (int) Math.ceil((Bukkit.getViewDistance() + 1) * Math.sqrt(2)));
 		packets.add(chunkradius);
 
+		ClientBoundPacketData enableCommandsPacket = ClientBoundPacketData.create(PEPacketIDs.SET_COMMANDS_ENABLED);
+		enableCommandsPacket.writeByte(1); // boolean true = enable commands
```

Comment:
writeBoolean?

Reply:
I thought we didn't have one... Thought I looked for it, didn't find it, looked how others did it, and settled for writeByte. I see now you're correct; will fix. I should probably also look at more places were we use writeByte currently (in a separate fix).